spec/models/subscribers/broker_digest_generator_subscriber_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Subscribers::BrokerDigestGeneratorSubscriber, "given active broker exists" do
+  let(:broker_agency_organization) { FactoryGirl.create(:benefit_sponsors_organizations_general_organization,:with_site,:with_broker_agency_profile)}
+  let!(:broker_agency_profile) { broker_agency_organization.broker_agency_profile }
+  let!(:person_broker) {FactoryGirl.create(:person,:with_work_email, :with_work_phone)}
+  let!(:broker) {FactoryGirl.create(:broker_role, benefit_sponsors_broker_agency_profile_id: broker_agency_profile.id, aasm_state: 'active',person: person_broker)}
+
+  subject { Subscribers::BrokerDigestGeneratorSubscriber.new }
+
+  describe "should generate broker digest" do
+
+    it "should notify broker digest published event" do
+      expect(subject).to receive(:notify) do |recipient, hash|
+        expect(recipient).to eq "acapi.info.events.brokers.broker_digest_published"
+        expect(hash[:return_status]).to eq "200"
+        expect(hash[:body]).to eq subject.instance_variable_get(:@body)
+      end
+      return_status = subject.work_with_params(nil, nil, {})
+      expect(return_status).to eq :ack
+    end
```

Comment:
A little confused about your spec here.

Shouldn't this do a little more?  For example, since you are getting the body, shouldn't you check some parts of it?

I.E.  is it a zip file?  What file names are in it?  How many entries does it have?

Reply:
specs added. Thanks @TreyE